### ISSUE TITLE
Album ArtworkUrlLarge is not the largest size

### ISSUE THREAD

**Issue by athornz** [TradeMe/PlayMe#32]
The image url returned the ArtworkUrlLarge field on the Album response is not the largest size available.

For example on an album with with 3 image sizes(64px, 300px, 640px) the ArtworkUrlLarge field returns the image that is 300px.

Can it return the largest size?

**Comment by fordprefect480** [comment]
Makes sense to me. I'll try and get to it, but you are also welcome to implement yourself :)
